packages/third-party/sourcemap-visualizer/sourcemapVisualizer.js

```diff
@@ -41,11 +41,11 @@ export default function renderSourcemap(source, sourcemap, url, document) {
       // Scan over the input
       while (true) {
         // Read a byte
-        if (i >= mappings.length) decodeError('Expected extra data');
```

Comment:
:speech_balloon: All this up here is from updating to the latest version of the file